support/client/lib/vwf/model/kineticjs.js

```diff
@@ -1657,8 +1665,12 @@ define( [ "module",
                     if ( kineticObj.getAttrs().hitGraphFromCache && kineticObj.isVisible() ) {
                         kineticObj.clearCache();
                         kineticObj.draw();
-                        kineticObj.cache();
-                        kineticObj.drawHitFromCache();
+                        if ( ( kineticObj.width() > kineticObj.getAttrs().minCachePixelSizeThreshold ) && 
+                             ( kineticObj.height() > kineticObj.getAttrs().minCachePixelSizeThreshold ) ) {
+                            kineticObj.cache();
+                            kineticObj.drawHitFromCache();
+                            kineticObj.draw();
```

Comment:
Looks like you weren't doing this extra `draw` before.  I'm thinking that since you're refreshing the hit graph here, that you shouldn't have to.